include/mbedtls/check_config.h

```diff
@@ -45,7 +45,7 @@
 #endif
 
 /* Fix the config here. Not convenient to put an #ifdef _WIN32 in config.h as
- * it would confuse config.pl. */
+ * it would confuse config.py. */
```

Comment:
I didn't check that specifically, but focused on compatibility over cleanliness, so I think it would still be an issue. I figured a complete rewrite and a behavior change didn't belong together.

Reply:
I am curious if we fixed this on accident by being cleaner with our regexes. Maybe not.